serverless/serverless.yml

```diff
@@ -6,14 +6,23 @@ package:
   individually: true
 
 plugins:
-  - serverless-bundle # Package our functions with Webpack
+  - serverless-webpack
   - serverless-offline
```

Comment:
does this fix that issue where we can't test stuff without having 'perfect' code?

Reply:
This solves that 'perfect' code issue since that bundle included babel within it. Mainly, I replaced the serverless-bundle since it seems quite complicated to use it with TypeScript.